docs/ai/internal-knowledge.md

```
@@ -18,6 +18,24 @@ GitHub repo metadata (description + topics) must match the current stack. When m
+1. Do **not** run bare `herdr` (that attaches the TUI). Use `herdr worktree`, `herdr agent`, or `herdr --skill`.
+2. `herdr worktree create` only creates a git worktree + **shell** pane. It does **not** start a coding agent.
+3. Required follow-up — parse `.result.root_pane.pane_id`, then start and prompt:
+
+```bash
+CREATE=$(herdr worktree create --cwd "$PWD" --branch feat/<name> --base origin/master --no-focus)
+PANE=$(printf '%s' "$CREATE" | jq -r '.result.root_pane.pane_id')
+herdr agent start feat-<short> --kind cursor --pane "$PANE"
+herdr agent prompt feat-<short> "<full task spec>" --wait --timeout 600000
+```
+
+4. Agent names: `[a-z][a-z0-9_-]{0,31}`, unique among live agents. Prefer `--kind` matching the user’s agent (Cursor → `cursor`, Grok → `grok`, etc.).
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)